.github/CODEOWNERS

```diff
@@ -2,19 +2,35 @@
 .github/                                @prekucki
 .github/workflows/rust.yml              @golemfactory/ya-integration
 
+# Activity
 core/activity/                          @golemfactory/exe-unit
 core/model/src/activity.rs              @golemfactory/exe-unit
+
+core/sgx/                               @golemfactory/exe-unit
+core/model/src/sgx.rs                   @golemfactory/exe-unit
+
 exe-unit/                               @golemfactory/exe-unit
 
+# Market
 core/market/                            @golemfactory/ya-market
 core/model/src/market.rs                @golemfactory/ya-market
 
+# Net
 service-bus/                            @golemfactory/ya-net
 core/net/                               @golemfactory/ya-net
 core/model/src/net.rs                   @golemfactory/ya-net
 
+# Payment
 core/payment-driver                     @golemfactory/ya-payment
 core/payment                            @golemfactory/ya-payment
 core/model/src/payment.rs               @golemfactory/ya-payment
 
+# Provider
 agent/provider                          @golemfactory/ya-provider
+golem_cli                               @golemfactory/ya-provider
+
+# Utils
```

Comment:
`ya-utils` is a strange name. At first I thought that this team is for for example `agreement-utils` etc. But I don't have better name...

Reply:
me neither :/